scripts/jekyll-watch.js

```diff
@@ -0,0 +1,48 @@
+var child_process = require('child_process');
+var chokidar = require('chokidar')
+
+// Run the given command with the given args in the given directory.
+//
+// Returns a Promise that resolves once the command is finished, or
+// rejects if the process errored or exited with a non-zero status code.
+function run(cwd, command, args) {
+  var cmdline = `${cwd}/${command} ${args.join(' ')}`;
+  return new Promise((resolve, reject) => {
+    console.log(`Running ${cmdline}.`);
+    const child = child_process.spawn(command, args, {
+      cwd: cwd,
+      stdio: 'inherit',
+    });
+    child.on('error', reject);
+    child.on('exit', code => {
+      if (code === 0) {
+        resolve();
+      } else {
+        reject(`${cmdline} exited with code ${code}!`);
+      }
+    });
+  });
+}
+
+function runJekyll() {
+  return run('/doi', 'jekyll', ['build', '--incremental'])
```

Comment:
Is any of the stuff happening in `serve-fast` still relevant here (besides `--incremental`)?

Reply:
I don't believe so. I'd like to remove the many different `*serve*` files and only have on Jekyll config. The host binding and such happens with docker and I don't know that we want to run the profiler/tracer every time we run the build. What do you think?

Reply:
Agreed

Reply:
Done in https://github.com/18F/doi-extractives-data/pull/2613